WordPress/gutenberg
# Add 'reusable' keyword to Pattern blocks

## What?
Fixes https://github.com/WordPress/gutenberg/issues/52194.

## Why?
Reusable blocks were renamed to Pattern blocks. We can help users adjust to this change by having pattern blocks appear when one searches the inserter for "reusable".

## How?
Adds "reusable" to the `keywords` for the Pattern block and to the virtual block definitions built by `getInserterItems` for each reusable block.

## Testing Instructions
1. Go to the post editor.
2. Add some blocks.
3. Select some blocks and press Options in the block toolbar.
4. Select _Create pattern/reusable block_.
5. Open the inserter.
6. Search "reusable".
7. Your newly created pattern should appear.
